Project.toml

```diff
@@ -33,7 +33,7 @@ Tullio = "bc48ee85-29a4-5162-ae0b-a64e1601d4bc"
 
 [compat]
 Adapt = "^3"
-CUDA = "3"
+CUDA = "3.0.0 - 3.3.6"
```

Comment:
```suggestion
CUDA = "~3.3"
```

Reply:
Or will that exclude v3.1 and 3.2?

Reply:
I think so. That's why I went with the range. But I'm not sure. Maybe "~3.0, ~3.1, ~3.2, ~3.3"? Not sure if that'll work.

Reply:
I read through it and can't be sure. I think the range is good enough for now. Hopefully in the next couple of weeks a next CUDA version will be tagged anyway. What do you think @navidcy?

I bumped a patch release.

Reply:
Why not pin to a specific version?

Reply:
@glwagner, range is OK. We had a range before anyways. Note that `Manifest.toml` pins a specific version.

@tomchor, I'll try to up to CUDA from v3.3.4 to v3.3.6 here. If the tests are OK then we should merge. Otherwise we should go back to  v3.3.4 and restrict the compat entry even more!

Reply:
@navidcy That's a good point! I don't think I actually tested with 3.3.6

@glwagner I think doing a range (instead of a specific version) adds some flexibility that might be necessary for users using other packages (which, in turn, might have restrictions CUDA restrictions of their own)

Reply:
> Note that Manifest.toml pins a specific version.

Right, that's why I figured making it specific was the safest option.